experimental/framework/tinylicious-client/src/test/ResettableTinyliciousClient.ts

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { TinyliciousClient } from "..";
+
+export class ResettableTinyliciousClient extends TinyliciousClient {
```

Comment:
I don't think this is a viable approach to the problem.  Without some means to coordinate between multiple users over the current state of the config, there's no way for a caller to understand if resetting is an appropriate action or who else they may affect by doing so.

I think the policy should be "If you want to be robust to scenarios that use multiple configs, do not use the statics but instead instantiate your own instance."

Reply:
Even if this is a test-only class?  You're right we definitely don't want people using this irl, and it's not getting exported in any way.

Reply:
Agree with Helio here. We are keeping it local to the testing folder for the sake of testing init() with and without port number specification.

Reply:
as per team decision, we will be removing static from TinyliciousClient. This implementation will be removed after refactoring.